environment-static-dev-win.yml

```diff
@@ -4,11 +4,12 @@ channels:
 dependencies:
   - vs2019_win-64
```

Comment:
This is not up to date either, even libmamba is built on a more recent vs, right?